Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
```

Comment:
I'm happy to release this tutorial under a creative commons license if you would like to reuse it.